goss-testing/vars/variables-ncn.yaml

```diff
@@ -163,3 +163,32 @@ storage_fs_labels:
 pit_node: false
 # false by default--overriden when vshasta is deployed
 vshasta: false
+
+
+pre_checks:
```

Comment:
The variables you are adding should probably all be prefaced with `iuf_`, except possibly the `bootprep` one. Otherwise it is not very obvious to someone looking at this file what they are related to.

Reply:
except bootprep we renamed all variables